go-fuzz/hub.go

```diff
@@ -116,6 +118,24 @@ func newHub(metadata MetaData) *Hub {
 			ro.intLits = append(ro.intLits, []byte(lit.Val))
 		}
 	}
+	if *flagDict != "" {
+		ro.strLits = nil // Discard existing tokens
+		fileName := *flagDict
+		log.Printf("reading token list from \"%v\"\n", fileName)
+		dictData, err := ioutil.ReadFile(fileName)
+		if err != nil {
+			log.Fatalf("could not read tokens from \"%v\": %v\n", fileName, err)
+		}
+		textData := string(dictData)
+		splits := strings.Split(textData,"\n")
+		for S := range splits {
```

Comment:
Will be shorter with `for _, token := range splits`.
Or even shorter with `for _, token := range strings.Split(textData,"\n")`.